azure-pipelines.yml

```diff
@@ -151,8 +149,11 @@ stages:
       - ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
         - job: Linux
           pool:
-            name: NetCore1ESPool-Svc-Public
-            demands: ImageOverride -equals Build.Ubuntu.1804.Amd64.Open
+            ${{ if eq(variables['System.TeamProject'], 'public') }}:
+              vmImage: ubuntu-18.04
+            ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+              name: NetCore1ESPool-Svc-Internal
+              demands: ImageOverride -equals Build.Ubuntu.1804.Amd64
```

Comment:
Looking at the `- ${{ if or(eq....` above, this job only runs on public CI anyway. We don't need to change this section.

Reply:
I changed my mind :) Let's keep it this way for completeness.